src/core/ui/markdown/WrapperMarkdown.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactMarkdown, { Options as ReactMarkdownOptions } from 'react-markdown';
+import gfm from 'remark-gfm';
+import rehypeRaw from 'rehype-raw';
+import emoji from 'remark-emoji';
+import components from './components';
+import PlainText from './PlainText';
+import { MarkdownOptions, MarkdownOptionsProvider } from './MarkdownOptionsContext';
+
+const allowedNodeTypes = ['iframe'] as const;
```

Comment:
I see we were already allowing iframes in the markdown, but is that really necessary?